### ISSUE TITLE
[HC] [Northrend] 

### ISSUE THREAD

**Issue by Boralamt** [dalaranwow/dalaran-wow#5643]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The current behavior is that some characters don't get the gem bag when the boss is defeated. They no longer get the gem bad from the cache of the ley guardian. The item never appears in the cache in the first place, although I have seen people in the same group being able to loot it.
**Expected behaviour**: Tell us what should happen instead
This gem bag should be lootable by all 5 party members.
**Steps to reproduce the problem**:

1. Queue up for a heroic Oculus.
2. Complete the instance by killing the Ley-Guardian.
3. Attempt to loot the Cache.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

The Oculus ID: 540

**Comment by Byssarod** [comment]
The Cache of the Ley-Guardian can be found in the Cache of Eregos after defeating Eregos in the Oculus on Heroic mode if and only if the player was selected to complete the dungeon randomly.

http://wowwiki.wikia.com/wiki/Cache_of_the_Ley-Guardian

**Comment by Golganeth** [comment]
Can't reproduce, all eligible players can loot the Cache from the correct object. Please provide valid evidence.

> Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
